app/routes/posts.js

```diff
@@ -0,0 +1,13 @@
+const postRoutes = require('express').Router();
+const multer  = require('multer')
+const upload = multer({ dest: 'uploads/' })
+const postController = require('../controllers/postController')
+
+
+postRoutes.post('/create_post', upload.single('mediaFile'), async (req, res) => {
```

Comment:
should be **post** is of create_post. The method explains the operation.

Reply:
you mean the api should be /post, right?